sormas-api/src/main/resources/strings.properties

```diff
@@ -313,7 +313,7 @@ errorNoPopulationDataLocations=No population data was found for the following lo
 errorNoPopulationDataFound=There is no population data available. Switching to absolute data view
 errorFormIdPopulationAgeGroup=Both "Form Id" and "Population Age Group" options are set 
 errorExternalSurveillanceToolNonCoronavirusCase=Could not send the selected cases to the reporting tool because the case %s is not a %s case.
-errorExternalSurveillanceToolCaseNotOwned=Could not send the selected cases to the reporting tool because your SORMAS instance is not the owner of the case %s.
+errorExternalSurveillanceToolCaseNotSharable=Could not send the selected cases to the reporting tool because. Please check if the case %s is editable and it is allowed to be be shared with external reporting tool.
```

Comment:
This error message is gramatically not correct and looks like it could be simplified.